src/Augurk/Scripts/angular-bootstrap-toggle.js

```diff
@@ -0,0 +1,177 @@
+(function () {
```

Comment:
Angular 1 feels so dirty to me these days ;).

Reply:
Deal with it.